integration_tests/test_search.py

```diff
@@ -1239,31 +1433,73 @@ def test_danielle_steel(self):
             ]
         )
 
-    def test_nabokov(self):
+    def test_primary_author_with_coauthors(self):
```

Comment:
These tests, especially test_narrator, check our new ability to search contributors with different roles.